common/common.cpp

```diff
@@ -151,13 +151,17 @@ bool gpt_params_parse_ex(int argc, char ** argv, gpt_params & params) {
             std::replace(arg.begin(), arg.end(), '_', '-');
         }
 
+        bool arg_found = false;
```

Comment:
Btw, to avoid this `arg_found` everywhere, we should move this if-chain in a `bool` function and return early on success

Reply:
I'll submit another PR.